Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -162,3 +162,46 @@
   - type: Item
     size: 5
     sprite: Objects/Weapons/Melee/e_cutlass.rsi
+
+- type: entity
+  name: Double Bladed Energy Sword
+  parent: EnergySword
+  id: EnergySwordDouble
+  description: Syndicate Command Interns thought that having one blade on the energy sword was not enough. This can be stored in pockets.
+  components:
+  - type: Wieldable
+    wieldTime: 1
+  - type: EnergySword
+    litDamageBonus:
+        types:
+            Slash: 6.5
+            Heat: 6.5
+            Structural: 6
```

Comment:
for the heat and slash, is it alright if I round up the heat and slash to 8 instead?

Reply:
See how many clicks it'll take with current vs regular esword and then see how many it will with it at 8.

Reply:
Sorry for the delay, went and got some food, anyways I did some math and assuming there correct (I fucken hope so) this is the change to 8 would be like. I also added in the heat and slash damage of 9 and 10 to see what it would be like.

Double Esword at 7.5 with an attack speed of 1.6 per second:
- Per swing: 15, DPS of 24, 7 swings to crit
- Per crit swing: 22.5, DPS of 36, 5 swings to crit

Double Esword at 8 with an attack speed of 1.6 per second:
- Per swing: 16, DPS of 25.6, 7 swings to crit
- Per crit swing: 24, DPS of 38.4, 5 swings to crit

Double Esword at 9 with an attack speed of 1.6 per second:
- Per swing: 18, DPS of 28.8, 6 swings to crit 
- Per crit swing: 27, DPS of 43.2, 4 swings to crit 

Double Esword at 10 with an attack speed of 1.6 per second:
- Per swing: 20, DPS of 32, 5 swings to crit 
- Per crit swing: 30, DPS of 48, 4 swings to crit 

Esword with an attack speed of 1:
- Per swing: 30, DPS of 30, 4 swings to crit
- Per crit swing: 60, DPS of 60, 2 swings to crit

Reply:
Put it at 9. If it's too much we can always reduce it later.

Reply:
done